### PR TITLE
BUGFIX: "indices" does not shuffling

### PR DESCRIPTION
**Pull request by amir-saniyan**
`tf.keras.random.shuffle` returns random elements and does not apply in-place

### CODE REVIEW

**Review by fchollet — COMMENTED** [review]
LGTM, thanks -- can you also update the md and ipynb files with the same?

**Comment by amir-saniyan** [comment]
> LGTM, thanks -- can you also update the md and ipynb files with the same?

md and ipynb files updated.

**Review by fchollet — APPROVED** [review]
LGTM, thanks!
